### ISSUE TITLE
Status of library?

### ISSUE THREAD

**Issue by georgwaechter** [bjjb/chromaprint.js#1]
Hey,

i'm wondering what is the status of the library? I wanted to experiment with it, but it seems the library is not finished yet. I did not find any methods as documented in the README. Can you list what is missing to make this working? The released npm package in version 0.1 seems to be empty (beside the package.json).

I would like to start a small experiment together with the web audio API. If i understand it correctly, i could let the browser decode (https://developer.mozilla.org/de/docs/Web/API/AudioContext/decodeAudioData) and analyse (https://developer.mozilla.org/de/docs/Web/API/AnalyserNode) the audio and feed the results (https://developer.mozilla.org/en-US/docs/Web/API/AnalyserNode/getFloatFrequencyData) somehow into this library, right?

Best

Georg

**Comment by zebateira** [comment]
You can download the lib and use it as is. It's just not published.
@bjjb any chance of fixing the package.json and publish it?

**Comment by bjjb** [comment]
Yup, but as @georgwaechter mentioned, the README doesn't reflect the status. I'll update the docs, and release a v0.0.1 on NPM later this week, then close this issue. As it stands, the chroma part works fine, but I want it to have a swappable resampling library.

**Comment by serhiikushch** [comment]
@bjjb, any updates?

**Comment by ebuildy** [comment]
I don't find the method `calculateFingerprint` to use it in a browser? Thanks you.

**Comment by vidhu** [comment]
Any updates? When are you planning to publish this library?

**Comment by bjjb** [comment]
Phew, it's been a while. :-) Since one of the goals of this project was to allow audio-fingerprinting in the browser (as well), it's a good time to resurrect it. I'll work on it this weekend. Stay tuned.

**Comment by alexfreska** [comment]
Really excited for your updates to this library 👍 publishing to npm would be very helpful

Thank you for your work!

**Comment by tvhdev** [comment]
also looking forward to an update, i need this badly for my educational music project. Thank you for your effort and time!

**Comment by joaovct** [comment]
> also looking forward to an update, i need this badly for my educational music project. Thank you for your effort and time!

After 3 years maybe you don't need it anymore, but recently I found me on the same problem. I tried this lib and another one, but none worked for me. So I created a new one, implementing the AcoustID Chromaprint. For me the best part is no need to install Chromaprint library manually. All you need is to install the package and use it on your NodeJS project.

https://github.com/joaodjtr/gimme-the-song/
